src/utils/__tests__/fuzzySearch-test.js

```diff
@@ -71,4 +71,15 @@ describe('util/fuzzySearch', () => {
     test('should return false when search is empty', () => {
         expect(fuzzySearch('', 'foo bar')).toBe(false);
     });
+
+    test('should return false when search/content are not a string', () => {
```

Comment:
We can probably remove these checks since we have the types in fuzzySearch.js. Since test files skip flow checks, the lines below don't have any flow errors.